translate/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Add a license header after the `<?xml>` line and before the `<project>` line. 

Optionally, you can also delete the `<?xml>` line and make the `<project>` line just `<project>`. See: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/appengine/helloworld-new-plugins/pom.xml